content/experiments/haunted-house.md

```diff
@@ -0,0 +1,295 @@
+---
+thumbnail: /haunted-house-demo.png
+title: Haunted house
+slug: haunted-house
+author: jaime-bboyjt
+description: The classical threejs-journey haunted house, done with TresJs
+tags: ['PointLight', 'fog', 'useTexture']
```

Comment:
add a `threejs-journey` tag, would be nice to filter in the future and send it to Bruno